src/somesy/cli/sync.py

```diff
@@ -97,6 +115,8 @@ def sync(
             cff_file=cff_file,
             no_sync_pyproject=no_sync_pyproject,
             pyproject_file=pyproject_file,
+            no_sync_codemeta=no_sync_codemeta,
+            codemeta_file=codemeta_file,
```

Comment:
COuld you update line 125-127 to print new cli inputs too?

Reply:
I was planning to... resulting in some serious refactoring :D but consider this done